### DISCUSSION TITLE
Why did Next.js choose pnpm over Yarn Berry or other package managers?

### DISCUSSION THREAD

**Discussion by jest-won** [vercel/next.js#74086] [App Router]
Why did Next.js choose pnpm over Yarn Berry or other package managers?

**Comment by icyJoseph** [comment]
#37259

I remember they used to have `yarn`, but switched to `pnpm`, relatively recently (Next.js has been around for a while...)

As per that PR I linked there, it looks like `pnpm` was producing faster builds, by reducing install size, and better dependency de-duplication. The most notable gain was that CI environments ran faster.
